Basic-Car-Maintenance/Shared/Settings/ViewModels/AuthenticationViewModel.swift

```diff
@@ -67,6 +67,7 @@ final class AuthenticationViewModel {
             if let user = Auth.auth().currentUser {
                 self.user = user
                 AnalyticsService.shared.setUserID(user.uid)
+                UserDefaults.shared.set(user.uid, forKey: "userID")
```

Comment:
we probably don't want to save it here cause it should be secure... is it possible to save to keychain instead?
also I'll look around how we should do this

Reply:
This has been removed. I was able to get the widget to respect the current user with FirebaseAuth + Keychain. For some reason, I couldn't wrap my head around the problem initially.